docs/Development/ReleaseNotes.md

```diff
@@ -107,6 +107,45 @@ This is a major update from VIC 4. The VIC 5.0.0 release aims to have nearly ide
 
 ------------------------------
 
+## VIC 4.2.c [![DOI](https://zenodo.org/badge/doi/10.5281/zenodo.35302.svg)](http://dx.doi.org/10.5281/zenodo.35302)
+
+Source code is available here: [![VIC.4.2.c](https://img.shields.io/badge/VIC-4.2.c-blue.svg)](https://github.com/UW-Hydro/VIC/releases/tag/VIC.4.2.c)
+
+**Release date: (December 12, 2015)**
+
+This is a bugfix update from 4.2.b.
+
+!!! Note "Note: Final Release of VIC 4 Development Track"
+	This is the last release of the VIC Version 4 development track.  The next release will be VIC.5.0 and will include backward incompatible changes.
+
+#### Bug Fixes:
+
+1.  Documented how VIC 4.2 needs user to specify veg_lib and veg_param files when OUTPUT_FORCE = TRUE. ([GH#305](https://github.com/UW-Hydro/VIC/issues/305))
+
+Prior to release 4.2, a user could run VIC in OUTPUT_FORCE mode with only a soil parameter file and forcing files.  This functionality is now broken as of release 4.2 and will not be fixed.  Users must either supply veg_lib and veg_parameter files (which the user is likely to have anyway) or use the standalone forcing disaggregator under cevelopment for use with release 5.0.  The documentation was updated to describe this issue as of release 4.2.c.
```

Comment:
Are you able to preview these locally?  It would help keep the guess and check to a minimum. Looking [here](https://github.com/tbohn/VIC/blob/update_5.0_release_notes_for_4.2.c/docs/Development/ReleaseNotes.md#bug-fixes-1), it doesn't seem like this format is working (number is reset after each paragraph).  I think you need one or two spaces to start the paragraph for the indentation to work out right.  

Docs for how build the VIC docs locally are here: https://github.com/tbohn/VIC/blob/update_5.0_release_notes_for_4.2.c/docs/readme.md